Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/BayesQuasi.py

```diff
@@ -763,12 +759,12 @@ def _read_ql_file(self, file_name, nl):
                 # SIGIK
                 line = next(line_pointer)
                 amp = AMAX * math.sqrt(math.fabs(line[0]) + 1.0e-20)
-                block_amplitude_e.append(amp)
+                block_amplitude_e.append(abs(amp))
 
                 # SIGFK
                 line = next(line_pointer)
                 FWHM = 2.0 * HWHM * math.sqrt(math.fabs(line[0]) + 1.0e-20)
-                block_FWHM_e.append(FWHM)
+                block_FWHM_e.append(abs(FWHM))
```

Comment:
How can `block_FWHM` be negative here unless `HWHM` is negative? Does this hint at another problem?

Reply:
I've only found that `block_height_e` can be negative sometimes, so that probably hints at a problem in the quasielasticbayes package. I then added `abs()` for the FWHM and amplitude just to be sure they were always positive in the workspace. Maybe I should avoid using `abs()` for the FWHM, amplitude and height until I have time to properly investigate inside quasielasticbayes?

Reply:
I'm OK with it - when quasielasticbayes gets fixed an extra `abs` or two will be pretty harmless. I'll approve when CI passes